lib/kb_ea_utils/kb_ea_utilsImpl.py

```diff
@@ -502,7 +502,7 @@ def run_Fastq_Multx(self, ctx, params):
         except Exception as e:
             raise ValueError('Unable to get ReadsUtils Client' +"\n" + str(e))
         try:
-            if input_reads_obj_type == "KBaseFile.PairedEndLibarary":
+            if input_reads_obj_type == "KBaseFile.PairedEndLibrary":
```

Comment:
I presume there couldn't have been a test checking this path or it would have failed, so a test should be added.